content/docs/user-guide/contributing/core.md

```diff
@@ -42,11 +42,12 @@ Get the latest development version. Fork and clone the repo:
 $ git clone git@github.com:<your-username>/dvc.git
 ```
 
-Make sure that you have Python 3 installed. Version 3.7 or higher is required to
-run style checkers on pre-commit. On macOS, we recommend using `brew` to install
-Python. For Windows, we recommend an official
+Make sure that you have Python 3.7 or higher installed. On macOS, we recommend
+using `brew` to install Python. For Windows, we recommend an official
 [python.org release](https://www.python.org/downloads/windows/).
 
+> ℹ️ Note that `pip` version 20.3+ is required.
```

Comment:
I think it's 20.3 due to https://www.pygit2.org/install.html but if it's another version lmk.

Reply:
That should be correct I think